resources/views/hr/application/index.blade.php

```diff
@@ -46,167 +45,153 @@
+                href= /{{ Request::path() }}?status={{ config('constants.hr.status.no-show.label') }}{{$query_filters}}>
+                    <sup class = "application-menu-options-title fz-20">
                         {{$noShowApplicationsCount+$noShowRemindedApplicationsCount}}
-                    </span>
+                    </sup>
+                    <span class="d-inline-block h-24 w-24">{!! file_get_contents(public_path('icons/exclamation-octagon-fill.svg')) !!}</span>
+                    <h5 class="no-show">No Show</h5>
                 </a>
             </li>
-            <li class="nav-item">
-                <a class="nav-item nav-link d-flex align-items-center {{ $status === config('constants.hr.status.rejected.label') ? 'active bg-info text-white' : 'text-info' }}"
-                    href= /{{ Request::path() }}?status={{ config('constants.hr.status.rejected.label') }}{{$query_filters}}>
-                    <i class="fa fa-times-circle"></i>&nbsp;
-                    Closed
-                    <span
-                        class="ml-1 d-inline-block px-2 py-0 {{ $status === config('constants.hr.status.rejected.label') ? 'active bg-white text-info' : 'bg-info text-white' }}"
-                        style="border-radius: 20px;font-size: 12px;font-weight: 700;">
-                        {{$rejectedApplicationsCount}}
-                    </span>
-                </a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-item nav-link d-flex align-items-center {{ $status === config('constants.hr.status.sent-for-approval.label') ? 'active bg-info text-white' : 'text-info' }}"
-                    href= /{{ Request::path() .'?status='. config('constants.hr.status.sent-for-approval.label')}}{{$query_filters}}>
-                    <i class="fa fa-clock-o"></i>&nbsp;{{ config('constants.hr.status.sent-for-approval.title') }}
-                    <span
-                        class="ml-1 d-inline-block px-2 py-0 {{ $status === config('constants.hr.status.sent-for-approval.label') ? 'active bg-white text-info' : 'bg-info text-white' }}"
-                        style="border-radius: 20px;font-size: 12px;font-weight: 700;">
+            <li id="list-styling">
+                <a id="job-application-listings" class="{{ $status === config('constants.hr.status.sent-for-approval.label') ? 'job-application-status' : '' }} btn"
+                href= /{{ Request::path() .'?status='. config('constants.hr.status.sent-for-approval.label')}}{{$query_filters}}>
+                    <sup class = "application-menu-options-title fz-20">
                         {{$sentForApprovalApplicationsCount}}
-                    </span>
+                    </sup>
+                    <span class="d-inline-block h-24 w-24">{!! file_get_contents(public_path('icons/clipboard.svg')) !!}</span>
+                    <h5 class="toapprove">To Approve</h5>
```

Comment:
Please correct the naming convention of class
```suggestion
                    <h5 class="to-approve">To Approve</h5>
```

Reply:
Also if it is not used anywhere, then you can remove it